src/exercises/day-2/luhn.md

```diff
@@ -21,10 +21,8 @@ function:
 
 
 ```rust
-// TODO: remove this when you're done with your implementation.
-#![allow(unused_variables, dead_code)]
-
-{{#include luhn.rs:luhn}}
+pub fn luhn(cc_number: &str) -> bool {
+    println!("{cc_number}");
```

Comment:
Please just add this line below the include we had before. That way we keep the connection between the exercise and the solution.

Reply:
Sure, does it make sense to add a new function to the original luhn.rs file and refer to it in this file or just add the line back in as is? The commit I just made does the latter.

Reply:
Hehe, this comment seems very similar to what we have in the other PR.

This reminds me of something I've been thinking about: can you try to avoid doing similar changes in different PRs? That is, please put up one PR for each conceptual change. Fewer and bigger PRs would make life easier for me as a reviewer.

Based purely on the `cchiw-patch-11` branch name, I assume you're doing these changes in the GitHub client? If so, that probably means that you haven't seen them live using `mdbook serve`?

Reply:
Yeah, I tried two _different_ approaches in the PRs because I wanted the reviewer to see the options. I think it totally make sense to just work on one PR at a time if it helps with the load. No problem.

I'll just work on this one, which has the line added back in as you requested, but now has a build error.

Reply:
> * Ah, yeah I wanted to show the _different_ approaches. Moving forward I'll just do one CL at a time to lower the load. Thanks for bringing this up.

Thanks! What you cannot know is that I review many of these CLs while commuting to work... so I'm often doing this on my phone. This means that I have little context and it's hard for me to compare between different PRs. So when I saw that you had several different PRs with different approaches, I put them back on the "for later" queue.

Reply:
Got it. I consolidated the open PRs. Last night found more places to remove the silence warnings and included them here. It's ready for review, but no rush.